docs/bots-guide.md

```diff
@@ -71,8 +71,8 @@ just need to write the `<my-bot>.py` script and put it into `zulip_bots/bots/<my
 You need:
 
 * An account in an organization on a Zulip server
-  (e.g. [chat.zulip.org](https://chat.zulip.org) or
-  yourSubdomain.zulipchat.com, or your own development server).
+  (e.g. [chat.zulip.org](https://chat.zulip.org),
+  yourSubdomain.zulipchat.com, or http://localhost:9991).
```

Comment:
Nice! What do you think about keeping the text `your own development server`, but linking it to this address?

Reply:
yep, makes sense